libpod/common/common.go

```diff
@@ -36,11 +36,14 @@ func GetCopyOptions(reportWriter io.Writer, signaturePolicyPath string, srcDocke
 }
 
 // GetSystemContext Constructs a new containers/image/types.SystemContext{} struct from the given signaturePolicy path
-func GetSystemContext(signaturePolicyPath string) *types.SystemContext {
+func GetSystemContext(signaturePolicyPath, authFilePath string) *types.SystemContext {
 	sc := &types.SystemContext{}
 	if signaturePolicyPath != "" {
 		sc.SignaturePolicyPath = signaturePolicyPath
 	}
+	if authFilePath != "" {
+		sc.AuthFilePath = authFilePath
```

Comment:
Isn't the default for sc.AuthFilePath == ""?

Reply:
yes, but when the user sets `outfile` or `infile`, it is saved in the systemContext and is used in the config package for the credentials.

Reply:
Right so the check for if the authfilepath != "" is useless.  You can just always say

 +		sc.AuthFilePath = authFilePath

Reply:
agreed, fixed.